plyer/platforms/android/qrbarcodereader.py

```diff
@@ -0,0 +1,21 @@
+from jnius import cast
+from jnius import autoclass
+from plyer.facades import QrBarcodeReader
+
+PythonActivity = autoclass('org.renpy.android.PythonActivity')
+Intent = autoclass('android.content.Intent')
+Uri = autoclass('android.net.Uri')
+
+class AndroidQrBarcodeReader(QrBarcodeReader):
+
+    def _scan(self):
+
+        intent = Intent()
+        intent.setAction(Intent.ACTION_VIEW)
+        intent.setData(Uri.parse('http://www.onlinebarcodereader.com'))
```

Comment:
Ok then, let's think zxing could work cross-platform. How do you intend to make it work on Apple hardware, which is pretty much ObjC/Swift? You'll still need to compile the Java code somewhere. There surely is a C/Cython library that can handle the things easier. Depending on how good you search it might already have a Python wrapper.